drivers/CoreIOExpander/include/external/MCP23017.h

```diff
@@ -0,0 +1,59 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <cstdint>
+
+namespace leka::mcp23017 {
+
+namespace pin {
+
+	constexpr inline uint16_t PA0 = 0x0001;
```

Comment:
je préfère la forme:

```suggestion
	inline constexpr auto PA0 = uint16_t {0x0001};
```